packages/back-end/src/services/features.ts

```diff
@@ -120,17 +133,21 @@ function generateVisualExperimentsPayload({
         phase?.savedGroups
       );
 
+      const urlRedirects = v.urlRedirects;
```

Comment:
If no urlPatterns for a URL redirect, exclude from the visual experiments for the payload